src/fetch.js

```diff
@@ -2,14 +2,20 @@ import gu from 'koa-gu'
 import co from 'co'
 import fileManager from './fileManager'
 import program from 'commander'
+import AWS from 'aws-sdk'
+
+// setup aws credentials
+gu.init({www:false});
+var credentials = new AWS.SharedIniFileCredentials(gu.config.aws_profile);
```

Comment:
can we use `const`?

```suggestion
const credentials = new AWS.SharedIniFileCredentials(gu.config.aws_profile);
```

Reply:
we do not have CI/CD now to deply apps on visuals ec2 so will keep it as was before, that is mainly code move